lib/crypto/secp256k1/secp256k1.go

```diff
@@ -36,6 +37,16 @@ type PublicKey struct {
 	key ecdsa.PublicKey
 }
 
+// SignVerify verify signature given pubkey and msg
+func SignVerify(pubkey, sig, msg []byte) (bool, error) {
```

Comment:
Can you change this to use `publicKey, signature, message []byte` for readability? Thanks!